docs/source/jupyter/import_export.ipynb

```diff
@@ -0,0 +1,275 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "boxed-professional",
+   "metadata": {},
+   "source": [
+    "# Tutorial: Import and export"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "operational-honey",
+   "metadata": {},
+   "source": [
+    "In this tutorial we will be covering the import and export capabilities of OSP-core. The utility functions that provide these functionalities are `import_cuds` and `export_cuds`, respectively."
```

Comment:
A link to try the tutorial in Binder is missing.

Con: the is no "file system" in Binder, so the user cannot directly see what has been exported. Nevertheless, the python `open` statement will just work as expected. Maybe add a comment under the Binder link: _"If you use Binder, there is no "filesystem view", so you will not be able to upload files or see what files you exported. However, the Python `open` statement will work as usual, so you may use it to read the files that you exported or create RDF files from a string."_.

Reply:
Disclaimer not needed, you are using `cat` and `more`.